packages/flutter/lib/src/widgets/container.dart

```diff
@@ -182,6 +182,8 @@ class DecoratedBox extends SingleChildRenderObjectWidget {
 /// [padding] (e.g. borders in a [BoxDecoration] contribute to the [padding]);
 /// see [Decoration.padding].
 ///
+/// ## Sample code
+///
```

Comment:
@gspencergoog The analyzer shard says this is deprecated, but it looks terrible without it (looks like it's in the subsection rather than a separate thing):

https://master-api.flutter.dev/flutter/widgets/Container-class.html

Any advice?

Reply:
I see.  The analyzer check is there to teach people to use the `{@tool ...}` syntax (that you're already using) instead of the old way.  You can just use a different heading text, e.g. "## Example code".  It won't be inconsistent, since nothing is using "## Sample code" anymore.

Alternatively, we could take out the analyzer check now that it's been a while since the transition, but I'd kind of like to keep it a little longer, just to catch when people forget.

Reply:
maybe we can make the check be silent if it's followed by the magic?

Reply:
Well it turns out there's a bunch of places we're not even analyzing right now where we still use `## Sample code`. Oops.

Reply:
There's also a bunch of places where we use \`\`\`dart. For example, look at the RenderBox docs. I'm not really sure what to do about cases like this.

Reply:
Yeah, I don't know what to do about the ```dart cases, since those are often such small or "out of context" samples that there's not enough context to evaluate them for correctness. We'd have to provide the context (imports, variable declarations, etc.), and I'm not sure how to provide that and not clutter up the comment with lots of weird syntax.

Where does the `## Sample code` still appear? I thought I had eliminated all of those in `packages/flutter`.

Reply:
packages/flutter_test, packages/flutter_driver, etc.